src/core/server/server.api.md

```diff
@@ -1928,6 +1928,8 @@ export type SharedGlobalConfig = RecursiveReadonly_2<{
 // @public
 export interface UiSettingsParams {
     category?: string[];
+    // Warning: (ae-forgotten-export) The symbol "DeprecationSettings" needs to be exported by the entry point index.d.ts
+    deprecation?: DeprecationSettings;
```

Comment:
Minor, but `DeprecationSettings ` should be exported

Reply:
https://github.com/elastic/kibana/pull/55020